src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -592,6 +592,10 @@ private void ExecuteClusterTransaction()
                     maxIndex = command.Index;
                 }
                 ClusterWideTransactionIndexWaiter.SetAndNotifyListenersIfHigher(maxIndex);
+
```

Comment:
Can you put both in a single method?

Reply:
At first, I considered creating a dedicated class to centralize the decision about the ID structure.
This would also allow us to reuse the notification key and ID instead of recreating them repeatedly.
However, I felt it might be overkill in this context.
If you agree it’s too much, I’ll simplify it by creating just a function to dismiss the notification.